syntax = "proto3";
package image;

service ImageClassify {
    rpc ClassifyImage (ImageClassifierRequest) returns (ImageClassifierResponse){}
}

message ImageClassifierRequest {
    bytes image = 1;
}

message ImageClassifierResponse {
    enum State {
        Error = 0;
        Ok = 1;
    }
    optional State state = 1;
    int64 class = 2;
}